afm-backend/controller/userController.py

```diff
@@ -22,4 +23,25 @@ def login():
     db.commit()    
     access_token = create_access_token(identity=username, expires_delta=datetime.timedelta(minutes=60))
     db = None
-    return jsonify(access_token=access_token)
\ No newline at end of file
+    return jsonify(access_token=access_token)
+
+@user_blueprint.route("/reset_password", methods=["PUT"])
+@jwt_required()
+def reset_password():
+    db = Database()
```

Comment:
Move all the db logic to userRepository

Then add the method in userDomain

And then call that from here

Reply:
Cool